docs/howto/connection/gcp.rst

```diff
@@ -32,7 +32,13 @@ There are three ways to connect to Google Cloud using Airflow.
 
 1. Use `Application Default Credentials
    <https://google-auth.readthedocs.io/en/latest/reference/google.auth.html#google.auth.default>`_,
-   such as via the metadata server when running on Google Compute Engine.
+   such as via the metadata server when running on Google Compute Engine or the GKE metadata server
```

Comment:
Can you move it to a separate section or paragraph below the text? I think this will make it easier to understand this list and its purpose. If we have a one-line explanation, we can put it in place, but explanations on many lines should be clearly separated.